src/styles/blocks/header.scss

```diff
@@ -0,0 +1,81 @@
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+
+  &__logo {
+    height: 40px;
+  }
+
+  &__nav .nav__hamburguer:first-child {
+    display: flex;
+    background: url(/src/images/icon-burger-menu.svg) no-repeat 50%;
+    background-size: contain;
+    transition: all 0.3s;
+    width: 32px;
+    height: 32px;
+
+    @media (min-width: 830px) {
+      display: none;
+    }
+  }
+}
+
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: none;
+    list-style: none;
+    text-transform: uppercase;
+
+    @media (min-width: 830px) {
+      display: flex;
+    }
+
+    &:target {
+      display: flex;
+    }
+  }
+
+  &__item {
+    position: relative;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    text-decoration: none;
+    transition: 0.3s;
+
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 60px;
+
+    &:hover {
+      color: $blue-color;
+    }
+  }
+}
+
+.is-active {
+  color: $blue-color;
+}
+
+.is-active::after {
```

Comment:
Could be nested into `.is-active` [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=mixins%2C%20functions%2C%20etc.-,Use%20nesting)